ios/Sources/GutenbergKit/Sources/Views/BlockInserter/BlockInserterSectionView.swift

```diff
@@ -24,6 +38,9 @@ struct BlockInserterSectionView: View {
                     .frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
WDYT about making the semantics of this element a heading? It could be convenient when navigating via VoiceOver.